vnext/Microsoft.ReactNative/ReactNativeAppBuilder.cpp

```diff
@@ -17,8 +18,12 @@ using namespace winrt::Microsoft::UI;
 namespace winrt::Microsoft::ReactNative::implementation {
 ReactNativeAppBuilder::ReactNativeAppBuilder() {
   m_host = winrt::make<winrt::ReactNative::implementation::ReactNativeHost>();
```

Comment:
Why is the host still created here? You're already storing everything else in the ReactNativeWin32App, just let it exist there.